raiden/blockchain/events.py

```diff
@@ -92,10 +92,13 @@ def get_contract_events(
         if filter_ is not None:
             filter_.uninstall()
 
-    return [
-        translator.decode_event(event['topics'], event['data'])
-        for event in events
-    ]
+    result = []
```

Comment:
This seems like a good change but I would suggest doing it the proper way for all events as is also suggested by [this](https://github.com/raiden-network/raiden/issues/450) issue. We need to get the block number from the JSON RPC and remove it from all events that we have it as an argument.